Emby.Server.Implementations/ScheduledTasks/Tasks/AudioNormalizationTask.cs

```diff
@@ -195,7 +195,8 @@ await foreach (var line in reader.ReadAllLinesAsync(cancellationToken))
 
                 if (match.Success)
                 {
-                    return float.Parse(match.Groups[1].ValueSpan, CultureInfo.InvariantCulture.NumberFormat);
+                        await process.WaitForExitAsync(cancellationToken);
```

Comment:
A timeout for waiting for the process to stop should be included here. I am thinking of 5 sec